docs/container-set-template.md

```diff
@@ -116,3 +116,11 @@ Example B: Lopsided requests, e.g. `a -> b` where `a` is cheap and `b` is expens
 Can you see the problem here? `a` only has small requests, but the container set will use the  total of all requests. So it's as if you're using all that GPU for 10h. This will be expensive.
 
 Solution: do not use container set when you have lopsided requests.
+
+## Container Set Retries
```

Comment:
we should have something here stating this is available from v3.3
https://github.com/argoproj/argo-workflows/commit/808c561f1c4a56668c32caa69be5b0441d372610

and give a little example would be good.

Reply:
OK，I add a example and version.Thanks!